app/src/main/res/values/strings.xml

```diff
@@ -109,6 +109,14 @@
     <string name="default_content_country_title">Default content country</string>
     <string name="service_title">Service</string>
     <string name="content_language_title">Default content language</string>
+    <string name="peertube_instance_url_title">PeerTube instances</string>
+    <string name="peertube_instance_url_summary">Set your favorite peertube instances</string>
+    <string name="peertube_instance_url_help">Find the instances that best suit you on https://joinpeertube.org/instances#instances-list</string>
```

Comment:
Just realized that the link needs to be an untranslatable resource. Otherwise people can change it while translating and route users to a different website.

Reply:
Fixed in https://github.com/TeamNewPipe/NewPipe/pull/2913/